packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -154,11 +154,15 @@ abstract class CachedArtifact {
     return stream.pipe(file.openWrite());
   }
 
+  /// The artifact file on disk.
   File get file =>
       File(p.join(cache.getArtifactDirectory(fileName).path, fileName));
 
+  /// Used to validate that the artifact was fully downloaded and extracted.
+  File get stampFile => File('${file.path}.stamp');
```

Comment:
Minor but I was under the impression that a stamp file was conventionally used to maintain a timestamp. Maybe there’s a better name here?

Reply:
Yeah, I was uncertain about this. I could write a timestamp to this file, but it seems a bit silly to do so if we never use it for anything.

Reply:
Looks like ninja stamp files are empty too.  It's just a "we finished task X" is all it's marking.  Then when you next wonder if you should run task X you can check to see last time it was finished and compare if it's inputs (in this case, presumably just the version of flutter or version of shorebird has changed since that time?

Reply:
Eric looked and said ninja does something similar so seems reasonable to me 👍